torch/csrc/autograd/python_variable.h

```diff
@@ -51,3 +51,5 @@ inline const at::Tensor& THPVariable_Unpack(THPVariable* var) {
 inline const at::Tensor& THPVariable_Unpack(PyObject* obj) {
   return THPVariable_Unpack(reinterpret_cast<THPVariable*>(obj));
 }
+
+THP_API c10::impl::PyInterpreter* getPyInterpreter();
```

Comment:
What's the reason for the `THP_API` instead of `TORCH_API`?

Reply:
it's the one for stuff in torch_python, which this file is in